lib/dce.rb

```diff
@@ -57,9 +57,9 @@ def docker_compose_file
     unless @compose_file
       dir = Dir.pwd()
       while dir != "/"
-        file = File.join(dir, 'docker-compose.yml')
-        if FileTest.exists?(file)
-          @compose_file = file
+        files = Dir[dir + '/docker-compose.{yml,yaml}']
+        if files.length > 0 and FileTest.exists?(files.first)
```

Comment:
In Ruby we use `!files.empty?`, but the cannonical implementation looks more like:

```ruby
@compose_file = Dir[dir + '/docker-compose.{yml,yaml}'].first
```

`[].first` returns `nil` which is false, so the loop carries on. Which does docker-compose pick if there's more than one, btw?

Reply:
It will return `.yml` before `.yaml` (it matches from left to right). This is what docker-composer also does.

I  have pushed and alternative implementation where I put the first on the Glob as you suggests and also drops the usage of FileTest.exists (if the name is returned by the glob I guess that is prove enough).